source/frontend/src/features/properties/map/acquisition/detail/AcquisitionDocumentsTab.tsx

```diff
@@ -16,12 +16,14 @@ const AcquisitionDocumentsTab: React.FunctionComponent<IAcquisitionDocumentsTabP
         parentId={acquisitionFileId}
         relationshipType={DocumentRelationshipType.ACQUISITION_FILES}
       />
-      <DocumentListContainer
-        title="Activity Documents"
-        parentId={acquisitionFileId}
-        relationshipType={DocumentRelationshipType.ACQUISITION_FILE_ACTIVITIES}
-        disableAdd
-      />
+      {false && (
```

Comment:
this should be disabled by claims, ie something like: hasClaim(claims.VIEW_ACTIVITY);
then when we can add/remove those claims without making a code change to show/hide this.